test/service/test_response.py

```diff
@@ -63,6 +63,7 @@ def test_key_search_files_response(self):
                         {
                             "organ": ["pancreas"],
                             "organPart": ["islet of Langerhans"],
+                            "selectedCellType": [],
```

Comment:
Since `[]` is the default value, this is insufficient proof that the `selectedCellType` property is actually propagated all the way from the contribution, through aggregate to the service response.